home/models.py

```diff
@@ -585,6 +588,7 @@ def submit_whatsapp_template(self, previous_revision):
             self.whatsapp_template_name,
             self.whatsapp_template_body,
             sorted(self.quick_reply_buttons),
+            self.get_whatsapp_image(),
```

Comment:
Lines 576-579, we're checking if the body, buttons, or the "is_whatsapp_template" property has changed. Maybe we should add there to check if the image has changed (or one was added)? Otherwise if you edit a message to add an image, no template will be submitted because nothing else changed.